components/resources/wallet_strings.grdp

```diff
@@ -1026,4 +1026,7 @@
   <message name="IDS_BRAVE_WALLET_PENDING_BALANCE_DESCRIPTION" desc="Available balance tooltip description">A pending change in your wallet balance.</message>
   <message name="IDS_BRAVE_WALLET_TOTAL_BALANCE_DESCRIPTION" desc="Total balance tooltip description">Your available funds plus any not-yet-confirmed transactions.</message>
   <message name="IDS_BRAVE_WALLET_UNAVAILABLE_BALANCES" desc="Some balances may be unavailable info bar">Some balances may be unavailable</message>
+  <message name="IDS_BRAVE_WALLET_RISK_OF_LOSS_ACTION" desc="Warning that the pending action may result in loss of funds">Unsafe action with risk of loss</message>
+  <message name="IDS_BRAVE_WALLET_UNTRUSTED_REQUEST_WARNING" desc="Warning that the pending request may be malicious">This request appears unstable or malicious, and may leave you vulnerable to theft. Do not continue unless you trust the app.</message>
```

Comment:
Is "unstable request" a term a user would understand?

Reply:
Not one that I've normally seen - maybe "high risk" instead

Reply:
Any suggestions @minkyeongshin ?

Reply:
@mkarolin , text has been updated